src/components/DatePicker.tsx

```diff
@@ -9,13 +9,18 @@ export type DatePickerProps = {
 };
 
 export default function DatePicker({ onRangeChange }: DatePickerProps) {
-  const [dateRange, setDateRange] = useState<DateRange>([null, null]);
+  const date24HrsAgo: Date = new Date(new Date().getTime() - (24 * 60 * 60 * 1000));
```

Comment:
we've already added dayjs as a project dependency so you might as well use it here :) 
```suggestion
  const date24HrsAgo: Date = dayjs().minus(1, 'day')
```